features/karaf/openhab-contrib/src/main/feature/feature.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+	Copyright (c) 2010-2019 Contributors to the openHAB project
+
+	See the NOTICE file(s) distributed with this work for additional
+	information.
+
+	This program and the accompanying materials are made available under the
+	terms of the Eclipse Public License 2.0 which is available at
+	http://www.eclipse.org/legal/epl-2.0
+
+	SPDX-License-Identifier: EPL-2.0
+
+-->
+<features name="${project.artifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+
+    <!-- Features for Java EE bundles -->
+
+    <feature name="openhab-jaxb" description="OpenHAB JAXB bundles" version="${project.version}">
+        <feature prerequisite="true">wrap</feature>
+        <bundle dependency="true">wrap:mvn:javax.activation/activation/1.1.1$Bundle-Name=Javax%20Activation&amp;Bundle-SymbolicName=javax.activation.activation&amp;Bundle-Version=1.1.1</bundle>
```

Comment:
Why not using the servicemix one (see PR about Java 11 compatibility on itests) and drop wrap?

Reply:
That's a problem. Due to KARAF-6077 (which is solved in Karaf 4.2.3, not in 4.2.2 which we currently use) the dependency to `org.apache.servicemix.specs/org.apache.servicemix.specs.activation-api-1.1/2.9.0` leads to an `UnsupportedOperationException` during fetaure verification while trying to uninstall `org.apache.servicemix.specs/org.apache.servicemix.specs.activation-api-1.1/2.5.0`.

Reply:
If it's just feature validation we could fix it by bumping Karaf in OHC. It doesn't need to be in sync with the version used in the distro.

Reply:
We would need to change it in openhab2-addons, but 'm not sure if it is verification only. The information in the issue is very rare.

Reply:
I think we should be able to use the servicemix bundles now we updated to Karaf 4.2.4 in openHAB and OHC also uses it for tooling now https://github.com/openhab/openhab-core/pull/740 is merged. :+1:

Reply:
Done